website/docs/kubernetes/configuring-editing-kube-object.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 12
+title: Managing objects
+description: Creating and updating a kubernetes object
+keywords: [podman desktop, podman, deploying, objects, kubernetes]
+tags: [managing-kubernetes, objects]
+---
+
+# Managing Kubernetes objects
+
+With Podman Desktop, you can enhance your Kubernetes management experience by using the features that are part of the Kubernetes explorer. As a developer, you can easily transition from containers to Kubernetes and deploy a complete Kubernetes environment with necessary objects. You have the ability:
```

Comment:
`With Podman Desktop, you can enhance your Kubernetes management experience by using the features that are part of the Kubernetes explorer.` feels like a run-off sentence, shorten it maybe? With Podman Desktop you can control your Kubernetes cluster or something similar?

Reply:
`As a developer, you can easily transition from containers to Kubernetes and deploy a complete Kubernetes environment with necessary objects.` is a repeat of the previous sentence

Reply:
Changed it to: With Podman Desktop, you can easily transition from containers to Kubernetes and deploy a complete Kubernetes environment with necessary objects. As a developer, you have the ability to:
- Manage your application resources visually.....